### DISCUSSION TITLE
vc-trail-polyline效果

### DISCUSSION THREAD

**Discussion by 987763485** [zouyaoji/vue-cesium#519] [General]
我想实现一个vc-trail-polyline效果，找不到任何文档和案例

**Comment by zouyaoji** [comment]
就是自定义一个材质。着急可以自己弄一个。
老的参考：
https://github.com/zouyaoji/vue-cesium-v2/blob/master/src/exts/materialProperty/PolylineTrailMaterialProperty.js
新的参考:
https://github.com/zouyaoji/vue-cesium/blob/dev/packages/shared/extends/materials/VcLineFlowMaterialProperty.ts

新的用法：
https://zouyaoji.top/vue-cesium/#/zh-CN/component/datasources/vc-datasource-custom


```
{
          position: [113, 28.5, 400],
          wall: {
            positions: [112.5, 28, 1000, 113, 28, 1000, 113, 29, 1000, 112.5, 29, 1000, 112.5, 29, 1000],
            material: {
              fabric: {
                type: 'VcLineFlow',
                uniforms: { image: 'https://zouyaoji.top/vue-cesium/images/textures/arrow.png', color: 'yellow', repeat: { x: 30, y: 1 }, speed: 10 }
              }
            }
          }
        }
```

**Comment by zouyaoji** [comment]
我下个版本集成一下
